src/main/scala/io/github/kiemlicz/shelm/HelmPlugin.scala

```diff
@@ -30,7 +30,16 @@ object HelmPlugin extends AutoPlugin {
     lazy val repositories = settingKey[Seq[ChartRepository]]("Additional Repositories settings")
     lazy val shouldUpdateRepositories = settingKey[Boolean]("Perform `helm repo update` at the helm:prepare beginning")
     lazy val chartSettings = settingKey[Seq[ChartSettings]]("All per-Chart settings")
+    lazy val downloadedChartsCache = settingKey[Option[File]](
+      "Directory in which plugin will search for charts before downloading them"
+    )
+    lazy val numberOfChartVersionsToKeep = settingKey[Int](
+      "How many versions of single chart should be kept in cache after cache cleaning"
+    )
 
+    lazy val cleanChartsCache = taskKey[Option[Int]](
```

Comment:
There should be an option to clear all, without providing exact number of charts.  
I was thinking that maybe `clean` task could do this but no, cache should be invalidated separately.